crest/Assets/Crest/Crest/Scripts/Helpers/Helpers.cs

```diff
@@ -142,6 +142,22 @@ public static void CreateRenderTargetTextureReference(ref RenderTexture texture,
             RenderTargetIdentifierXR(ref texture, ref target);
         }
 
+        /// <summary>
+        /// Creates an RT with an RTD if it does not exist or assigns RTD to RT. This prevents reference leaks.
+        /// </summary>
+        public static void SafeCreateRenderTexture(ref RenderTexture texture, RenderTextureDescriptor descriptor)
+        {
+            // Do not overwrite reference or it will create reference leak.
+            if (texture == null)
+            {
+                texture = new RenderTexture(descriptor);
+            }
+            else
+            {
+                texture.descriptor = descriptor;
```

Comment:
Do we need to call create or does this apply the new descriptor settings automatically

Reply:
We'd could call Create when ready or let Unity create the texture when is used

Reply:
I've updated the summary. The links are useful to understand when Create is needed. I think it is there mainly for reading/writing pixels in C# maybe. But generally can get away without it.